api/v1/db/oracledb/onbase-dao.js

```diff
@@ -0,0 +1,107 @@
+const appRoot = require('app-root-path');
+const createError = require('http-errors');
```

Comment:
This package looks helpful for throwing different errors in modules without requiring `res` to be passed down to it. @nosnatef was asking about the best practice for handling this. Do you think this should go in the skeleton somewhere?

As an alternative, do you think we could use the `notFound` and `badRequest` functions in [errors/errors.js](./errors/errors.js) to build and reject (using `throw`) a jsonapi error? Then the path handler should just be able to send that error. The main reason for suggesting this is that `createError` expects a single error string but we may need to return a `400` response with multiple strings. This would require us to join the strings into a single string and then split them again in the path handler.

What are your thoughts on this?